src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -57,18 +70,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       parent = KeyFactory.createKey(Comment.ENTITY_KIND, parentId);
     }
 
+    User author = userService.getCurrentUser();
+    if (author == null) {
+      response.sendError(HttpServletResponse.SC_FORBIDDEN, "You are not logged in.");
+      return;
+    }
+
     String content = CharStreams.toString(request.getReader());
 
-    // TODO: Most of these values are default placeholders. Add real values as features are added.
     Entity commentEntity = new Entity(Comment.ENTITY_KIND);
     commentEntity.setProperty(Comment.PROP_LECTURE, lecture);
     commentEntity.setProperty(Comment.PROP_PARENT, parent);
+    // TODO: Add support for timestamped comments
     commentEntity.setProperty(Comment.PROP_TIMESTAMP, new Date(0));
-    commentEntity.setProperty(Comment.PROP_AUTHOR, "");
+    commentEntity.setProperty(Comment.PROP_AUTHOR, author);
     commentEntity.setProperty(Comment.PROP_CONTENT, content);
     commentEntity.setProperty(Comment.PROP_CREATED, new Date());
```

Comment:
Probably want to add a TODO for this as well?

Reply:
Actually `new Date()` creates a `Date` for the current time.  I'll have to check if there's a better way to make a Date for now in the google best practices

Reply:
Added suggested comment

Reply:
And switched to using java.time.Clock